src/styles.css

```diff
@@ -230,7 +230,7 @@
 
```

Comment:
- The accessibility of your portfolio looks great 😃, but we still have some issues when we zoom the font. Let's make sure that Text, images, and other content do not overlap.

<img width="802" alt="image" src="https://user-images.githubusercontent.com/77190203/149983456-8d9aa2c9-bf55-4eed-951a-b90de43d5cd9.png">

Reply:
thanks for reviewing , if you can help and give me some hints on how to do that i will be thankful

Reply:
Hi @souad988,
I would suggest not fixing the height of the elements. Try always using min-height, and assign `auto` for the height. `height: auto`

Reply:
ok thanks for your help